src/main/scala/com/typesafe/conductr/client/ConductRController.scala

```diff
@@ -179,15 +182,18 @@ class ConductRController(conductr: Uri, loggingQuery: Uri, connectTimeout: Timeo
   import context.dispatcher
 
   override def receive: Receive = {
-    case GetBundleInfoStream   => fetchBundleSource(sender())
-    case GetEventStream(b)     => fetchLoggingQuerySource[Event](sender(), b)
-    case GetLogStream(b)       => fetchLoggingQuerySource[Log](sender(), b)
-    case request: LoadBundle   => loadBundle(request)
-    case request: RunBundle    => runBundle(request)
-    case request: StopBundle   => stopBundle(request)
-    case request: UnloadBundle => unloadBundle(request)
+    case request: GetBundleInfoStream => fetchBundleSource(sender(), request)
+    case request: GetEventStream      => fetchLoggingQuerySource[Event](sender(), request.protocolVersion, request.bundleId)
+    case request: GetLogStream        => fetchLoggingQuerySource[Log](sender(), request.protocolVersion, request.bundleId)
+    case request: LoadBundle          => loadBundle(request)
+    case request: RunBundle           => runBundle(request)
+    case request: StopBundle          => stopBundle(request)
+    case request: UnloadBundle        => unloadBundle(request)
   }
 
+  private def protocolVersionPath(version: String): String =
+    if (version == "1.0") "" else "/v" + version
```

Comment:
That would mean creating extra routes in ConductR - also if we issue a `/v1.0` to ConductR 1.0 then it will fail.